src/index.js

```diff
@@ -138,6 +144,21 @@ const InlineSvgComponent = {
                 request.send();
             });
         },
+
+        /**
+         * Create or edit the <title> element of a SVG
+         * @param {Object} svg
+         */
+        setTitle(svg) {
+            const titleTags = svg.getElementsByTagName('title');
+            if (titleTags.length) { // overwrite existing title
+                titleTags[0].innerHTML = this.title;
```

Comment:
I'm thinking of replacing `.innerHTML` with `.innerText`. 
It will prevent XSS. 
On the other hand, it will disable inserting other tags or html entities, like `&nbsp;`. But titles are used for accessibility and screen readers. Usually they doesn't need these features.

Reply:
Ooh, good point. I'd go with `.innerText` instead.